docs/installation/standalone.mdx

```diff
@@ -0,0 +1,417 @@
+  daemon                	daemonsets.apps         	Describes daemonset services in Kubernetes.
+  cron-task             	cronjobs.batch          	Describes cron jobs that run code or a script to completion.
+  worker                	deployments.apps        	Describes long-running, scalable, containerized services
+  that running at backend. They do NOT have network endpoint
+  to receive external network traffic.
+  task                  	jobs.batch              	Describes jobs that run code or a script to completion.
+  webservice            	deployments.apps        	Describes long-running, scalable, containerized services
+  that have a stable network endpoint to receive external
+  network traffic from customers.
+  config-image-registry 	autodetects.core.oam.dev	Config information to authenticate image registry
+  k8s-objects           	autodetects.core.oam.dev	K8s-objects allow users to specify raw K8s objects in
+  properties
+  raw                   	autodetects.core.oam.dev	Raw allow users to specify raw K8s object in properties.
+  This definition is DEPRECATED, please use 'k8s-objects'
+  instead.
+  ref-objects           	autodetects.core.oam.dev	Ref-objects allow users to specify ref objects to use.
+  Notice that this component type have special handle logic.
+  ```
+
+</details>
+Export kubeconfig and list all built-in ComponentDefinition:
+
+### 3. Install VelaUX
+
+VelaUX is a dashboard for KubeVela. It is a web application that runs in your cluster. You can access it with your browser.
+This is optional if you don't use UI console of KubeVela.
+
+<Tabs className="unique-tabs" groupId="os">
+  <TabItem label="Linux/macOS" value="unix">
+
+  ```shell script
+  vela addon enable ~/.vela/addons/velaux
+  ```
+
+  </TabItem>
+  <TabItem label="Windows" value="win">
+
+  ```powershell
+  vela addon enable $HOME\.vela\addons\velaux
+  ```
+
+  </TabItem>
+</Tabs>
+
+<details>
+  <summary>expected output</summary>
+
+  ```
+  Addon: velaux enabled Successfully.
+  ```
+
+</details>
+
+By default, velaux didn't have any exposed port, you can view it by:
+
+```shell
+vela port-forward addon-velaux -n vela-system 8080:80
+```
+
+Choose `> local | velaux | velaux` for visit.
+
+:::note
+For more VelaUX options, refer to [VelaUX addon](../reference/addons/velaux) document for other advanced installation arguments.
+:::
+
+VelaUX needs authentication. The default username is `admin` and the password is **`VelaUX12345`**.
+
+It requires you to override with a new password for the first login, please make sure to remember the new password.
+
+### 4. Cleanup
+
+This command will clean up KubeVela controllers along with the Kubernetes cluster, refer to [the advanced guide](../platform-engineers/advanced-install#uninstall) for more detailed steps.
+
+```
+velad uninstall
+```
+
+## Install on a remote Linux server {#remote}
+
+This documents show you how to install KubeVela on a remote Linux server (e.g. aws EC2 or aliyun ECS) in standalone mode.
```

Comment:
Let's add some tips to explain why we don't support other OS for remote server.

Reply:
done